### PR TITLE
Move to Java 8 Time (EXPOSUREAPP-13147)

### PR DESCRIPTION
**Pull request by mtwalli**
Part of removing Joda-Time and moving completely to Java 8

## Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13147

### CODE REVIEW

File: src/test/kotlin/PlusTimeTest.kt [hunk 1]
```diff
@@ -8,55 +8,55 @@ import kotlin.test.assertFailsWith
 internal class PlusTimeTest {
     @Test
     fun `plus time in hours`() {
-        val firstDate = "2013-08-11T17:22:04.51+01:00"
```
Comment (chiljamgossow) [inline src/test/kotlin/PlusTimeTest.kt:11]: How come you changed this? This input should be allowed and work.

Comment (mtwalli) [reply src/test/kotlin/PlusTimeTest.kt:11]: the issue is not in the input , it is accepted. Joda-Time was not that accurate in this case. if you pass milli-seconds then the output also should be assessed including milii-seconds , I adjusted the tests now to reflect that

**Review by jurajkusnier — APPROVED** [review]
LGTM, fixes also [EXPOSUREAPP-13165](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13165)

File: src/test/kotlin/PlusTimeTest.kt (hunk 1 above)
Comment (chiljamgossow) [reply src/test/kotlin/PlusTimeTest.kt:11]: I had this in there resultDate.toString(ISODateTimeFormat.dateTimeNoMillis(), so dateTimeNoMillis() was doing the trick before. Input can be anything, just the output should not contain millis

Comment (chiljamgossow) [reply src/test/kotlin/PlusTimeTest.kt:11]: That's the reason the tests were set up like this.

Comment (mtwalli) [reply src/test/kotlin/PlusTimeTest.kt]: Fixed!

**Review by chiljamgossow — APPROVED** [review]
Nice, thanks!

**Review by chiljamgossow — APPROVED** [review]

**Pull request merged**